tests/README.md

```diff
@@ -1,5 +1,9 @@
 To use these tests run the following:
 
+The environment variables set by the pipeline need to be overwritten. The pipeline automatically overwrites the target host, therefore we need to unset them so we can deploy the pods within the correct environment. 
```

Comment:
This isn't relevant here - its specific to the pipeline environment. It should be detailed in the pipeline repo where you actually use it.